### PR TITLE
Add several unit tests for azure_controller_common.go and azure_controller_standard.go

### PR DESCRIPTION
**Pull request by nilo19**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Add several unit tests for azure_controller_common.go. and azure_controller_standard.go

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/legacy-cloud-providers/azure/azure_instances_test.go [hunk 1]
```diff
@@ -57,10 +58,25 @@ func setTestVirtualMachines(c *Cloud, vmList map[string]string) {
...
+				for i := 0; i < maxLUN; i++ {
+					dataDisks[i] = compute.DataDisk{Lun: to.Int32Ptr(int32(i))}
+				}
+				vm.StorageProfile.DataDisks = &dataDisks
 			}
-		}
 
-		store["rg"][nodeName] = vm
+			store["rg"][nodeName] = vm
+		}
```
Comment (feiskyer) [inline staging/src/k8s.io/legacy-cloud-providers/azure/azure_instances_test.go]: The `}` should be moved above `store["rg"][nodeName] = vm`

**Comment by feiskyer** [comment]
Need to run ./hack/update-bazel.sh locally to update the bazel files.

File: staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_standard_test.go [hunk 2]
```diff
@@ -0,0 +1,112 @@
+package azure
```
Comment (andyzhangx) [inline staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_standard_test.go]: you need a copy right header here.

**Review by andyzhangx — COMMENTED** [review]
pls always run following script before submitted code, that will save you lots of back and forth failure check
@ss104301 
```
#precheck before submit code
hack/verify-golint.sh
hack/verify-gofmt.sh

#fix bazel build issue if there is file name change in BUILD
hack/update-bazel.sh
```

File: staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_standard_test.go (hunk 2 above)
Comment (ss104301) [reply staging/src/k8s.io/legacy-cloud-providers/azure/azure_controller_standard_test.go]: Got it. THX andy.

**Pull request closed without merging**